apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -1136,18 +1239,35 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
   end
 
   describe "/transaction.create validation" do
-    @tag fixtures: [:alice, :more_utxos]
-    test "empty transaction without payments list is not allowed", %{alice: alice} do
+    @tag fixtures: [:alice, :more_utxos, :test_server]
+    test "empty transaction without payments list is not allowed", %{alice: alice, test_server: context} do
       alice_addr = Encoding.to_hex(alice.addr)
 
+      fee_response = %{
+        @str_tx_type => [
+          %{
+            "currency" => @default_fee_currency,
+            "amount" => 0,
```

Comment:
similar here, try with the default and see if it works. The fee amount should not be relevant here, at 1st glance. Maybe setting it to 0 was superfluous originally